src/main/java/org/example/Dbcon.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+ class DbCon {
+
+    public static Connection connection;
+    public static void conn(){
+        String url = "jdbc:mysql://localhost:3306/employee";
```

Comment:
Create a class as **StringConstants** and declare all string there. 

Example: 

```
class StringContants { 
    public static final String url = "jdbc:mysql://localhost:3306/employee";
    ....
}
```